Src/Plugins/Tests.Security/Validators/SEC101_020.DropboxAccessTokenTokenValidatorTests.cs

```diff
@@ -16,6 +16,9 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
 {
+    /// <summary>
+    /// Testing SEC101/020.DropboxAccessTokenValidator
```

Comment:
>DropboxAccessTokenValidator [](http://example.com/codeflow?start=27&length=27)

The name of this file is wrong, SEC101_020.DropboxAccessTokenTokenValidatorTests.cs, the word 'Token' is repeated.